### PR TITLE
DOC Linting

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-mfa/issues/531

Also adds in docs from totp + webauthn modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,65 +3,15 @@
...
-## Installation
-
-```sh
-composer require silverstripe/mfa
-```
-
-You should also install one of the additional multi-factor authenticator modules:
-
-* [silverstripe/totp-authenticator](https://github.com/silverstripe/silverstripe-totp-authenticator)
-* [silverstripe/webauthn-authenticator](https://github.com/silverstripe/silverstripe-webauthn-authenticator)
```
Comment (ChloeSartorelli) [inline README.md]: I think module standardiser requires an `Installation` section. We should keep this here.

File: docs/en/07_other-authenticators.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
 # Integrating with other authenticators
 
-**This version relates to Silverstripe CMS 4.x configuration.**
-
-If your project uses a non-standard authentication module, such as silverstripe/ldap, you will
+If your project uses a non-standard authentication module, such as [silverstripe/ldap](https://github.com/silverstripe/silverstripe-ldap), you will
```
Comment (ChloeSartorelli) [inline docs/en/07_other-authenticators.md]:
```suggestion
If your project uses a non-standard authentication module, such as [`silverstripe/ldap`](https://github.com/silverstripe/silverstripe-ldap), you will
```

File: docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md [hunk 3]
```diff
@@ -23,10 +21,11 @@ When logging in with the **Keep me signed in** option selected, a time-based coo
...
 In addition to remembering the device, your username and password will also be remembered.
 
-<div class="notice" markdown='1'>
-It is important that you only choose to use this option on a device you know to be secure. A secure device may be one that requires you to log in before being able to access the Internet, for example one within an office network, or a personal mobile device with a passcode.
+> [!IMPORTANT]
+> Only choose to use this option on a device you know to be secure. A secure device may be one that requires you to log in before being able to access the Internet, for example one within an office network, or a personal mobile device with a passcode.
 
 **A device shared between multiple people would not be considered secure and you should not use this option.**
 </div>
+```
```
Comment (ChloeSartorelli) [inline docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md]: These backticks look like they might be a mistake? And the `</div>` should be removed
```suggestion
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
totp and webauthn docs should stay where they are. You can add them as children of the mfa docs by setting the name to something like `docs--5--optional_features/mfa/totp` or as children of the "authenticators" directory by setting it to something like `docs--5--optional_features/mfa/authenticators/totp`


Please also add title medadata to all pages.
e.g
```text
---
title: Some Page Name
summary: A brief summary of what the page content is
icon: link
---
```
The icon and summary are optional.

File: README.md (hunk 1 above)
Comment (emteknetnz) [reply README.md]: Updated

File: docs/en/07_other-authenticators.md (hunk 2 above)
Comment (emteknetnz) [reply docs/en/07_other-authenticators.md]: Updated

File: docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md (hunk 3 above)
Comment (emteknetnz) [reply docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**